### ISSUE TITLE
[Windows] Unable to get top games from Steamspy

### ISSUE THREAD

**Issue by Gnomesenpai** [tpill90/steam-lancache-prefill#120]
Fresh download of 1.5.0

> SteamPrefill.exe prefill --top
> [2:07:21 AM] Starting login!
> [2:07:22 AM] Logged into Steam
> [2:07:22 AM] Steam session initialization complete!
> [2:07:23 AM] An unexpected error occurred while retrieving most played games from SteamSpy!  Popular games will be
> excluded from this prefill run.

**Comment by tpill90** [comment]
One possible alternative is to scrape the Steam stats page : https://store.steampowered.com/stats/steam-game-and-player-statistics

```csharp
using WebClient client = new WebClient();
client.DownloadFile("https://store.steampowered.com/stats/steam-game-and-player-statistics", @"Output.html");
```

**Comment by tpill90** [comment]
Another alternative is the new Steam Stats page : https://store.steampowered.com/charts/mostplayed

https://github.com/SteamDatabase/Protobufs/commit/4b9e2edb662ca62a63daad3595fc7aa23c1ee728
![image](https://user-images.githubusercontent.com/766513/191972907-370a38b7-3a5e-4637-a2c2-b5e1ed953d47.png)

**Comment by tpill90** [comment]
Replaced SteamSpy with the official Steam api, and looks to be working as expected: 
![image](https://user-images.githubusercontent.com/766513/200979595-da832b01-f205-45dd-92af-af70fa42cc51.png)

**Comment by tpill90** [comment]
@Gnomesenpai Would you mind giving this test build a shot? 
https://github.com/tpill90/steam-lancache-prefill/actions/runs/3433297573

**Comment by tpill90** [comment]
Released in [v1.8.0](https://github.com/tpill90/steam-lancache-prefill/releases/tag/v1.8.0)
